larq_compute_engine/mlir/python/converter.py

```diff
@@ -0,0 +1,79 @@
+from larq_compute_engine.mlir._graphdef_tfl_flatbuffer import (
+    convert_graphdef_to_tflite_flatbuffer,
+)
+
+from tensorflow.core.framework.types_pb2 import DataType
+from tensorflow.lite.python.util import (
+    get_grappler_config,
+    get_tensor_name,
+    run_graph_optimizations,
+)
+from tensorflow.python.eager import def_function
+from tensorflow.python.framework import convert_to_constants, dtypes
+from tensorflow.python.keras.saving import saving_utils
+
+
+def convert_keras_model(model):
```

Comment:
Should we add a comment to mention that this is derived from the `from_keras_model` function in [this file](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/python/lite.py) ? Then we know that updates to that function can be upstreamed into our function.

Reply:
I agree. Are you fine with doing this in a future PR? Then I'd like to also add support for different TensorFlow versions and add the relevant tests to CI as well.